src/Importify/Options.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE ApplicativeDo #-}
+
+-- | Command line options for Importify
+
+module Importify.Options
+       ( Command(..)
+       , SingleFileOptions(..)
+       , parseOptions
+       ) where
+
+import           Universum
+
+import           Options.Applicative (Parser, ParserInfo, argument, command, execParser,
+                                      fullDesc, help, helper, info, metavar, progDesc,
+                                      str, subparser)
+
+data Command
+    = SingleFile SingleFileOptions
+
+data SingleFileOptions = SingleFileOptions
+    { sfoFilename   :: !FilePath
+    -- ^ File to apply to tool to
+    }
+
+optionsParser :: Parser Command
+optionsParser =
+    subparser
+        (command
+            "file"
+            (info (helper <*> singleParser)
+                  (fullDesc <> progDesc "Importify a single file"))) <|>
+    singleParser
+
+
+singleParser :: Parser Command
+singleParser = do
+    sfoFilename <- argument str $
+        metavar "FILE" <>
+        help "File to importify"
+    return $ SingleFile SingleFileOptions{..}
```

Comment:
`pure` is better because `return` may not work with `-XApplicativeDo`. But that's not so important because I'm not even sure that `-XRecordWildCards` make `-XApplicativeDo` enabled.